manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanerThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+PackageLocale: de-DE
+Publisher: Luca-Alexander Thomas
+PublisherUrl: https://lucathomas.de
+PublisherSupportUrl: https://lucathomas.de/kontakt
+PrivacyUrl: https://lucathomas.de/datenschutzerklaerung/
+Author: Luca-Alexander Thomas
+PackageName: AutoHome-Autostart
+PackageUrl: https://github.com/luca-alexander-thomas/AutoHome
+License: GPL-3.0
+LicenseUrl: https://github.com/luca-alexander-thomas/AutoHome/blob/Autostart/LICENSE
+Copyright: 2021 by Luca-Alexander Thomas
+ShortDescription: AutoHome ist ein Autostart Programm welches Programme Startet wenn ein bestimmtes Laufwerk vorhanden ist. AutoHome ist Perfekt geeignet für Dock Setups Zuhause oder auf der Arbeit.
+Description: AutoHome ist ein Autostart Programm welches Programme Startet wenn ein bestimmtes Laufwerk vorhanden ist. AutoHome ist Perfekt geeignet für Dock Setups Zuhause oder auf der Arbeit.
+Moniker: AutoHome
+Tags:
+- Autostart

```

These should be lower case for consistency
```suggestion
Moniker: autohome
Tags:
- autostart
```